resources/[gameplay]/chat/webpack.config.js

```diff
@@ -1,11 +1,18 @@
 const HtmlWebpackPlugin = require('html-webpack-plugin');
+const path = require('path');
 const HtmlWebpackInlineSourcePlugin = require('html-webpack-inline-source-plugin');
-const VueLoaderPlugin = require('vue-loader/lib/plugin');
+const { VueLoaderPlugin } = require('vue-loader');
 const CopyPlugin = require('copy-webpack-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
 
 module.exports = {
     mode: 'production',
     entry: './html/main.ts',
+    output: {
+      filename: '[name].[contenthash:8].js',
```

Comment:
Having a content hash is a bit weird here... and wouldn't this break the fxmanifest.lua entry?

Reply:
hi, sorry please feel free to continue this PR if you wish. I no longer work around FiveM related code :)